### PR TITLE
Add general and pregnancy tutorial

### LINKED ISSUE #114: Pregnancy tutorial
Write a tutorial for the pregnancy module. Likely best completed after #109.

### LINKED ISSUE #127: Re-structure tutorials
Make general tutorials along with more module-specific ones. Include a table of contents and links to other resources.

### PR DESCRIPTION
**Pull request by pineapple-cat**
Addresses #127 and #114.

### CODE REVIEW

**Comment by pineapple-cat** [comment]
Pregnancy data variable candidates for change.

- [x] Rename: `fold_preg` (name could be less confusing); `rate_testanc_inc` (could use another underscore); `rate_birth_with_infected_child` (this is a probability, not a rate)
- [x] Make a new issue for: keeping `inc_cat` consistent across modules
- [x] Make a new issue for: move and rename `fold_tr_newp`

File: tutorials/pregnancy.md [hunk 1]
```diff
@@ -8,3 +8,19 @@ Welcome to the pregnancy module. This code deals with fertility, pregnancy, ante
...
+### Pregnancy Data Variables
+
+- *`can_be_pregnant`* - The proportion of women in a population that can become pregnant.
+- *`rate_want_no_children`* - The rate at which women stop wanting children over time.
+- *`date_pmtct`* - The year during which PMTCT begins (2004 by default).
+- *`pmtct_inc_rate`* - The rate at which the probability of receiving PMTCT increases per year.
+- *`fold_preg`* - A fertility factor that changes with age and affects pregnancy probability.
+- *`inc_cat`* - A demographic variable that affects population growth and pregnancy probability.
+- *`prob_pregnancy_base`* - The base probability of pregnancy (calculated when the pregnancy module is initialised).
+- *`rate_testanc_inc`* - An additive modifier to `prob_anc`.
```
Comment (mmcleod89) [inline tutorials/pregnancy.md]: *`rate_testanc_inc`* - The amount by which `prob_anc` increases per timestep (additive)

File: tutorials/pregnancy.md [hunk 2]
```diff
@@ -8,3 +8,19 @@ Welcome to the pregnancy module. This code deals with fertility, pregnancy, ante
 - `src/hivpy/pregnancy_data.py` - A class for storing data loaded from `pregnancy.yaml`.
 
 If there are any pregnancy-related variables you would like to change before running your simulation, please change them in `pregnancy.yaml`.
+
+### Pregnancy Data Variables
+
+- *`can_be_pregnant`* - The proportion of women in a population that can become pregnant.
+- *`rate_want_no_children`* - The rate at which women stop wanting children over time.
+- *`date_pmtct`* - The year during which PMTCT begins (2004 by default).
+- *`pmtct_inc_rate`* - The rate at which the probability of receiving PMTCT increases per year.
```
Comment (mmcleod89) [inline tutorials/pregnancy.md]: Multiplicative

**Comment by mmcleod89** [comment]
Let's add a short introduction to the model behind the module and how that works 
- Probabilisitically generates pregnancies based on number of partners etc. 
- Pregnancy is tracked by the date of pregnancy
- Do no model pregnancies not carried to term 
- Children do not become new people to be tracked in the model 
- ANC related testing at each trimester

**Review by mmcleod89 — APPROVED** [review]

**Pull request merged**